### PR TITLE
Add a test for resource commands.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

File: src/command_modules/azure-cli-resource/tests/test_custom.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    @classmethod
+    def tearDownClass(cls):
+        pass
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
```
Comment (troydai) [inline src/command_modules/azure-cli-resource/tests/test_custom.py]: None of the set up fixtures do anything. Please remove it.

Comment (troydai) [inline src/command_modules/azure-cli-resource/tests/test_custom.py]: None of the set up fixtures do anything. Please remove it.

File: src/command_modules/azure-cli-resource/tests/test_custom.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+    def test_merge_parameters(self):
```
Comment (troydai) [inline src/command_modules/azure-cli-resource/tests/test_custom.py]: Please add a more accurate name. Due to the current test layout, you'd better have the test method name to be unique so as to facilitate the parallel test run. For example, this method can be named to `test_resource_merge_parameters`.

File: src/command_modules/azure-cli-resource/tests/test_custom.py (hunk 1 above)
Comment (brendandburns) [inline src/command_modules/azure-cli-resource/tests/test_custom.py]: done.

File: src/command_modules/azure-cli-resource/tests/test_custom.py (hunk 2 above)
Comment (brendandburns) [inline src/command_modules/azure-cli-resource/tests/test_custom.py]: done.

**Comment by derekbekoe** [comment]
@troydai look good to you?

**Pull request merged**